dkg/src/epoch_manager.rs

```diff
@@ -89,24 +90,42 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
 
     fn process_rpc_request(
         &mut self,
-        _peer_id: AccountAddress,
-        _dkg_request: IncomingRpcRequest,
+        peer_id: AccountAddress,
+        dkg_request: IncomingRpcRequest,
     ) -> Result<()> {
-        //TODO
+        if Some(dkg_request.msg.epoch()) == self.epoch_state.as_ref().map(|s| s.epoch) {
+            // Forward to DKGManager if it is alive.
+            if let Some(tx) = &self.dkg_rpc_msg_tx {
+                tx.push((), (peer_id, dkg_request));
```

Comment:
why the key is ()? we use the peer_id for DDoS prevention

Reply:
fixed